bin/init

```diff
@@ -4,4 +4,4 @@
 source "$(dirname "${BASH_SOURCE[0]}")/_config.sh"
 
 # Run ansible-playbook
-_docker_run ansible-playbook init_project.yml --ask-vault-pass "$@"
+_docker_run ansible-playbook quickstart.yml --ask-vault-pass "$@"
```

Comment:
but shouldn't the bash command have the same name as the playbook if all it does is call it?

Reply:
Yes it's a good idea. I do this.